### ISSUE TITLE
Support for Office add-ins in group mailboxes

### ISSUE THREAD

**Issue by mailchecktool-admin** [OfficeDev/office-js#3839]
Hello,

We have been using an Outlook JavaScript add-in (ItemSend event) provided by Microsoft for our services. We launch and sell services using this add-in. I created this inquiry to confirm the specifications of the Office add-in.

According to the official documentation of Office add-ins, it is not supposed to be supported in the "microsoft365 group" folder in Outlook on the web. The documentation explicitly states that group mailboxes are not supported.

[Link to official documentation](https://learn.microsoft.com/en-us/office/dev/add-ins/outlook/outlook-add-ins-overview#mailbox-items-available-to-add-ins)

Until recently, we have verified that Office add-ins were not functioning in group mailboxes. However, around last month, the Office add-in started working. Does this mean that support for group mailboxes has been initiated for Office add-ins?

**Comment by timwan10** [comment]
What client version of Outlook are you using that has support for this? A Screenshot showing this working would help. I do not believe support has been enabled for group mailboxes, so this may be a bug.

**Comment by mailchecktool-admin** [comment]
@timwan10 
Thank you for your prompt response. 
The host version of Outlook on the web is as follows.
HostVersion: 20231103003.23
 Additionally, I will share a reproduction video.
 https://drive.google.com/file/d/1TyR5IJBLtteVbstW4AQqA0n_aWvDtu9W/view?usp=drive_link

**Comment by mailchecktool-admin** [comment]
@timwan10 
Have you had a chance to review the response I provided the other day? I have heard that there might be a possibility of a bug, so I would appreciate it if you could investigate it. Additionally, I have received an inquiry from one of my clients, and I need to provide an explanation. Please let me know if there is anything you are currently aware of that I can share with them.

**Comment by mailchecktool-admin** [comment]
@timwan10 
This issue has not been resolved yet. Add-ins are still running in the Microsoft 365 group. 
However, the official documentation for the Office add-in describes it as "not becoming active."


> However, add-ins are not activated if the current message item, in a compose or read form, is one of the following:
> ・
> ・
> In a [**group mailbox**](https://learn.microsoft.com/en-us/microsoft-365/admin/create-groups/compare-groups?view=o365-worldwide&preserve-view=true#shared-mailboxes), in a shared mailbox*, in another user's mailbox*, in an [archive mailbox](https://learn.microsoft.com/en-us/office365/servicedescriptions/exchange-online-archiving-service-description/archive-client-and-compliance-&-security-feature-details?tabs=Archive-features#archive-mailbox), or in a public folder.

I have a question regarding the statement you made about "support not being enabled." Does this mean that even though it appears that the add-in is running, there are situations where the add-in may not actually be executed, and emails could be sent without it running? If we send emails expecting the add-in to be executed and it doesn't happen, it could lead to serious problems.

Many users send emails from Microsoft 365 groups, and it would be helpful to receive support for cases involving the use of add-ins with Microsoft 365 groups. Could you please share your current stance or intention regarding this matter?

**Comment by ztcdsb** [comment]
@mailchecktool-admin Thanks for offering the video. This is expected, when you open a new compose from group mailbox, you are sending the email "From" your mailbox, not the group mailbox. 
But you shouldn't see the add-in when read the email

Your video recording is for OWA platform, I will also reach out to them to confirm the behavior

**Comment by mailchecktool-admin** [comment]
@ztcdsb 
Thank you for your response.
I see, understood.
As the issue has been resolved, I will close this ticket. Thank you very much.

**Comment by t-ariyoshi30907** [comment]
@exextoc 
I wanted to reach out with a question. Upon testing the issue here today, I confirmed that the problem has been resolved. Now, even when sending emails from the group mailbox, the add-in is functioning properly.

However, upon checking the official documentation, it still states that "group mailboxes are not supported."

Which is correct, my test results or the content of the documentation?
